.eslintrc.json

```diff
@@ -201,7 +203,9 @@
         "no-var": "error",
         "prefer-const": "error",
         "prefer-template": "error",
-        "unused-imports/no-unused-imports-ts": 2
+        "unused-imports/no-unused-imports-ts": 2,
```

Comment:
what does the 2 means? Is it a threshold or so?

Reply:
I don't know, was automatically added by the update migration.